### ISSUE TITLE
Unable to initialize REFPROP in MATLAB R2020a

### ISSUE THREAD

**Issue by sylviaff** [usnistgov/REFPROP-wrappers#325]
I have followed the instruction of REFPROP for MATLAB. I got some issues in initialize REFPROP.

```
>> RP = py.ctREFPROP.ctREFPROP.REFPROPFunctionLibrary('C:\Program Files (x86)\REFPROP')
NULL pointer returned from Python.
```
Previously, I installed REFPROP in D:\Program Files (x86)\REFPROP and tried : 
```
>> RP = py.ctREFPROP.ctREFPROP.REFPROPFunctionLibrary('D:\Program Files (x86)\REFPROP')
NULL pointer returned from Python.
```
Then I changed the installation to C disk.

### Versions

**REFPROP Version:** 10.0a
**Operating System and Version:** Windows 10
**Access Method:** Matlab R2020a

### Additional Information
The whole process:
```
>> pyversion

       version: '3.8'
    executable: 'D:\Program Files (x86)\Python\python.EXE'
       library: 'D:\Program Files (x86)\Python\python38.dll'
          home: 'D:\Program Files (x86)\Python'
      isloaded: 0

>> [v,e] = pyversion; system(['"', e, '" -m pip install --user -U ctREFPROP'])
Collecting ctREFPROP 
  Using cached ctREFPROP-0.8.2-py3-none-any.whl (19 kB) 
Installing collected packages: ctREFPROP 
Successfully installed ctREFPROP-0.8.2 

ans =

     0

>> RP = py.ctREFPROP.ctREFPROP.REFPROPFunctionLibrary('C:\Program Files (x86)\REFPROP')
NULL pointer returned from Python.
```

**Comment by ianhbell** [comment]
Nice issue report! Very clear.

**Comment by ianhbell** [comment]
When you moved REFPROP to the D: drive, did you also update the PATH environment variable to point to ``D:/Program Files (x86)/REFPROP``?  That is sometimes required, but I do like you are doing usually in Python and it works for me without changing the PATH variable.

MATLAB makes changes to the PATH variable, at least that was my experience some years ago.  Can you try to run your example directly in Python?

Do you really mean you have version 10.0a? What version is that?  Can you do calculations in the GUI?

**Comment by sylviaff** [comment]
> When you moved REFPROP to the D: drive, did you also update the PATH environment variable to point to D:/Program Files (x86)/REFPROP?  That is sometimes required, but I do like you are doing usually in Python and it works for me without changing the PATH variable.
> MATLAB makes changes to the PATH variable, at least that was my experience some years ago.  Can you try to run your example directly in Python?
> Do you really mean you have version 10.0a? What version is that?  Can you do calculations in the GUI?

Yes, I updated the Path environment. Unluckily, it didn't work. 
My research group purchased the version 10.0a, so yes, I can use this software. Calculations in the GUI can be successfully done.
One of my colleague used the same instruction as posted for MATLAB wrapper and succeeded in an earlier time. He helped me but also confused with my issue. So I came here for some help.
One good news is that after trying again and again, I can now recall refpropm in Matlab through .dll method. He helped me with this! I do agree with you that Python would work better. 
Many thanks for your reply!

**Comment by ianhbell** [comment]
Can you try to completely uninstall REFPROP, delete the REFPROP folder, search your computer for any other copies of REFPROP DLL, delete them, and then reinstall REFPROP 10?  Sometimes old DLL get stuck in odd places (e.g., c:\windows\system32 or something).  You can also try to pass the absolute path to the dll you want to load right into REFPROPFunctionLibrary.

Again, can you please confirm precisely what version of REFPROP you have? Is is the 10.0.0.0 release or 10.0.0.2 or an alpha or beta release?  10.0a is not a standard version.  Also, the GUI uses the 32-bit DLL, while your code uses the 64-bit DLL, another common source of confusion.

The refpropm method is crippled and will be removed at some point, so let's figure out what the issue is.

**Comment by EricLemmon** [comment]
Microsoft uses ghost directories for overly protected systems so that it appears that the file is in one place but in reality the files are put in your users directory, under the following hidden subdirectory:

AppData\Local\VirtualStore\Program Files\REFPROP

So, when you are searching for old Refprop files, be sure to search hidden and system directories as well, and delete the directory above if it exists.

**Comment by ianhbell** [comment]
I guess also the folder:

```
AppData\Local\VirtualStore\Program Files (x86)\REFPROP
```

if you use 64-bit OS

**Comment by sylviaff** [comment]
For now, I am quite confused and almost give up...
****
First, I followed your instruction:
uninstall REFPROP, delete the REFPROP folder
search refprop
AppData\Local\VirtualStore\
I didn't have the files as mentioned. Even the folder of VirtualStore is blank.

In this folder, AppData\Local\Temp\tpce74a74d_ef73_41e5_95f6_56c8d631bb3d
I deleted refprop_thunk_pcwin64.dll

Tried reinstall. The same errors as I posted. (NULL pointer returned from Python.)
Both C drive and D drive were tried.

I even unistalled Matlab, REFPROP and re-install again. It didn't work.

BTW, I checked the version, it is 10.0alpha.
